site/assets/scss/_component-examples.scss

```diff
@@ -251,7 +251,7 @@
 
 // Boosted mod
 .simple-list-example-scrollspy .active {
-  color: var(--#{$prefix}link-hover-color);
+  color: var(--bs-link-hover-color);
```

Comment:
It seems like there is also in `site\assets\scss\_anchor.scss`, `site\assets\scss\_clipboard-js.scss`, `site\assets\scss\_search.scss` and `site\assets\scss\_tarteaucitron.scss` where we need to change these values

Reply:
I was thinking about doing it in the next commits from Bootstrap, but maybe there won't be any regarding this topic. I don't remember so let's do it now; I'll play with the Boosted mods if needed. Done via https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/1817/commits/91ac6a9a007c8c72762231252ba411215ba9e8aa